client/src/pages/SearchResultsUser.js

```diff
@@ -11,6 +12,24 @@ function SearchResultsUser() {
   }
 
   const query = useQuery().get("query");
+  componentDidMount() {
+    fetch("https://codesandbox.io/s/react-router-query-parameters-mfh8p?from-embed")
+      .then(res => res.json())
+      .then(
+        (result) => {
+          this.setState({
```

Comment:
Also keeping in mind of the functional components, instead of setState, I would switch this to the useState that we've been setting up everywhere else, called something like `const [results, setResults] = useState(null)`.